src/app/static/src/app/store/modelRun/mutations.ts

```diff
@@ -51,8 +51,7 @@ export const mutations: MutationTree<ModelRunState> = {
     },
 
     [ModelRunMutation.RunStatusError](state: ModelRunState, action: PayloadWithType<Error>) {
-        state.errors.push(action.payload);
-        if (state.errors.length >= maxPollErrors) {
+        if (state.pollingCounter >= maxPollErrors) {
```

Comment:
I think we also need to reset pollingCounter as well don't we, for next time we want to run. Should do that in stopPolling, so it happens for all cases.